jkube-kit/config/image/src/main/java/org/eclipse/jkube/kit/config/image/ImageName.java

```diff
@@ -57,6 +57,8 @@ public class ImageName {
 
     private static final int REPO_NAME_MAX_LENGTH = 255;
 
+    private static final int MIN_DIGEST_LENGTH = 64;
```

Comment:
This can vary depending on the algorithm, for SHA256, it is 64. But for SHA512 it's 128.

Reply:
made the changes accordingly